arch/arm/core/aarch32/thread.c

```diff
@@ -112,6 +112,13 @@ void arch_new_thread(struct k_thread *thread, k_thread_stack_t *stack,
 #endif /* CONFIG_COMPILER_ISA_THUMB2 */
 #endif /* CONFIG_CPU_CORTEX_M */
 
+#if defined(CONFIG_CPU_AARCH32_CORTEX_R) \
```

Comment:
`!defined(CONFIG_CPU_CORTEX_M)` would make more sense in this case since this applies to both Cortex-A and Cortex-R.

Reply:
done